config/prometheus/prometheus-configmap.yaml

```diff
@@ -24,4 +24,4 @@ data:
       # scheme defaults to 'http'.
       metrics_path: '/prometheus'
       static_configs:
-      - targets: ['topic-gateway:80','topic-controller:80','event-dispatcher:80','spring-handler:80','greeter:80']
+      - targets: ['topic-gateway:80','topic-controller:80','function-controller:80','spring-handler:80','greeter:80']
```

Comment:
Makes me realize I missed topic-gateway here

Reply:
also I guess `spring-handler` is obsolete as well

cc @trisberg